tests/flytekit/unit/extend/test_agent.py

```diff
@@ -399,16 +399,3 @@ def sample_agents():
         name="ChatGPT Agent", is_sync=True, supported_task_categories=[TaskCategory(name="chatgpt", version=0)]
     )
     return [async_agent, sync_agent]
-
-
-@patch("flytekit.clis.sdk_in_container.serve.click.secho")
-@patch("flytekit.extend.backend.base_agent.AgentRegistry.list_agents")
-def test_print_agents_metadata_output(list_agents_mock, mock_secho, sample_agents):
-    list_agents_mock.return_value = sample_agents
-    print_agents_metadata()
-    expected_calls = [
-        (("Starting Sensor that supports task categories ['sensor']",), {"fg": "blue"}),
-        (("Starting ChatGPT Agent that supports task categories ['chatgpt']",), {"fg": "blue"}),
-    ]
-    mock_secho.assert_has_calls(expected_calls, any_order=True)
-    assert mock_secho.call_count == len(expected_calls)
```

Comment:
why are we removing this test?

Reply:
the test is failing because we use rich to print the metadata instead of secho. https://github.com/flyteorg/flytekit/blob/2b49bb3343c8480c388d1c81f340e2f42626e9f2/flytekit/clis/sdk_in_container/serve.py#L105